beproud/django/commons/middleware/debug.py

```diff
@@ -7,5 +7,5 @@ def process_response(self, request, response):
         if settings.DEBUG: 
             from django.db import connection
             for query in connection.queries:
-                print query
+                print(query)
```

Comment:
THX for response. Sure, I won't add a `form __future__ import *`.

BTW, are there any idea to determine whether to use `__future__` or any other workaround?
I want to avoid missing importation of `__future__`.